programs/bubblegum/program/src/error.rs

```diff
@@ -108,6 +108,8 @@ pub enum BubblegumError {
     StakingVoterAuthorityMismatch,
     #[msg("Invalid mining owner")]
     MiningOwnerMismatch,
+    #[msg("Cannot create pubkey from string")]
```

Comment:
Do we still need this error? Seems like a relict from times when Pubkeys were strings.